core/lib/dal/src/consensus_dal.rs

```diff
@@ -95,27 +91,55 @@ impl ConsensusDal<'_, '_> {
         Ok(())
     }
 
+    /// Fetches the range of miniblocks present in storage.
```

Comment:
These are blocks present in storage, empty range x..x indicates that x is the first block to be inserted. I'd rather leave it as it is for now. I'll apply your suggestion in case the current semantics turns out to be too implicit.